web/typescript/page/edit.ts

```diff
@@ -0,0 +1,9 @@
+export function checkMaxLength() {
+    [...document.querySelectorAll('textarea[maxlength]')].forEach((element) => {
+        const maxlength = element.getAttribute('maxlength');
+        element.removeAttribute('maxlength');
+        element.addEventListener('input', () => {
+            element.setCustomValidity(maxlength >= element.value.length ? '' : 'Your imput must be shorter than ' + maxlength + ' characters.');
```

Comment:
```suggestion
            element.setCustomValidity(maxlength > element.value.length ? 'Your input must be shorter than ' + maxlength + ' characters.' : '');
```

Reply:
Not sure but we might want to utilize [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) to format the maxlength number. @vednoc thoughts?

Reply:
Up to you. It would be a step towards internationalization, right?

Reply:
I would rewrite it like so:
```suggestion
            element.setCustomValidity(element.value.length > maxlength ? `Your input must be up to ${maxlength} characters.` : '');
```

Reply:
I like this more.